### ISSUE TITLE
Pip Windows cu121 not working

### ISSUE THREAD

**Issue by ghzeni** [pytorch/pytorch#115551]
### 🐛 Describe the bug

Upon selecting the following parameters to download from [pytorch.org](https://pytorch.org/)

![image](https://github.com/pytorch/pytorch/assets/15947263/c66d85c4-e2f7-4841-b981-b998d068862c)

I receive the following response:

```
C:\src> pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121

Looking in indexes: https://download.pytorch.org/whl/cu118
Collecting torch
  Could not find a version that satisfies the requirement torch (from versions: )
No matching distribution found for **torch**
```

Upon searching I came across the following solution:

[stackoverflow](https://stackoverflow.com/a/64584662) > [stackoverflow](https://stackoverflow.com/a/66723943) > [github issue](https://github.com/pytorch/pytorch/issues/54172#issuecomment-967309338) > [github issue](https://github.com/pytorch/pytorch/issues/52662)

Which I don't understand the reason it being closed since the command generated from the [download page](https://pytorch.org/get-started/locally/) is not currently working for windows.

What worked for me was the following response from the second stackoverflow link mentioned above:

> TL;DR run this command instead:
> ```
> pip install torch==1.8.0+cu101 torchvision==0.9.0+cu101 torchaudio===0.8.0 -f https://download.pytorch.org/whl/torch_stable.html
> ```

I'm in doubt as to why this issue was closed in the first place without being resolved. Am I missing something? Thanks.

### Versions

```
wget : The response content cannot be parsed because the Internet Explorer engine is not available, or Internet         Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again.             At line:1 char:1                                                                                                        + wget https://raw.githubusercontent.com/pytorch/pytorch/main/torch/uti ...                                             + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                     + CategoryInfo          : NotImplemented: (:) [Invoke-WebRequest], NotSupportedException                                + FullyQualifiedErrorId : WebCmdletIEDomNotSupportedException,Microsoft.PowerShell.Commands.InvokeWebRequestComman     d   
```

So much for windows 11 support I guess... IE does not exist anymore, please fix the `Please run the following and paste the output below.` prompt to run on WIN 11 aswell.

Actual command output:
```
Collecting environment information...
PyTorch version: 1.8.0+cu101
Is debug build: False
CUDA used to build PyTorch: 10.1
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 11 Pro
GCC version: Could not collect
Clang version: Could not collect
CMake version: Could not collect
Libc version: N/A

Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] (64-bit runtime)     
Python platform: Windows-10-10.0.21996-SP0
Is CUDA available: True
CUDA runtime version: Could not collect
CUDA_MODULE_LOADING set to:
GPU models and configuration: GPU 0: GeForce GTX 1050 Ti
Nvidia driver version: 456.71
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Architecture=9
CurrentClockSpeed=2808
DeviceID=CPU0
Family=205
L2CacheSize=1536
L2CacheSpeed=
Manufacturer=GenuineIntel
MaxClockSpeed=2808
Name=Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
ProcessorType=3
Revision=

Versions of relevant libraries:
[pip3] numpy==1.21.6
[pip3] torch==1.8.0+cu101
[pip3] torchaudio==0.8.0
[pip3] torchvision==0.9.0+cu101
[conda] _anaconda_depends         2023.09             py311_mkl_1
[conda] blas                      1.0                         mkl
[conda] mkl                       2023.1.0         h6b88ed4_46357
[conda] mkl-service               2.4.0           py311h2bbff1b_1
[conda] mkl_fft                   1.3.8           py311h2bbff1b_0
[conda] mkl_random                1.2.4           py311h59b6b97_0
[conda] numpy                     1.24.3          py311hdab7c0b_1
[conda] numpy-base                1.24.3          py311hd01c5d8_1
[conda] numpydoc                  1.5.0           py311haa95532_0
```

cc @seemethere @malfet @osalpekar @atalman @peterjc123 @mszhanyi @skyline75489 @nbcsm @vladimir-aubrecht @iremyux @Blackhex @cristianPanaite

**Comment by malfet** [comment]
To install pytorch-2.1 one needs Python 3.8 or newer, which is clearly stated [on the same page](https://pytorch.org/get-started/locally/#windows-python)
But you are using python-3.7, which is not supported. Please update your python runtime and try again
